Description: Add elpa metadata file
Author: Sean Whitton <spwhitton@spwhitton.name>
Forwarded: https://github.com/agda/agda/issues/2032

This file is needed for the elpa-agda2-bin binary package to build.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- /dev/null
+++ b/src/data/emacs-mode/agda2-mode-pkg.el
@@ -0,0 +1,2 @@
+(define-package "agda2-mode" "2.5.1"
+  "interactive development for Agda, a dependently typed functional programming language")
